asset-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Asset Store
+
+## Overview
+
+This example illustrates how to use [asset-store](https://kyma-project.io/docs/1.5/components/asset-store/) to store simple webpages.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kyma](https://kyma-project.io/docs/) as the target deployment environment
+
+## Details
```

Comment:
The [ README template](https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/example_README.md) for examples does not include the ##Details section. Please remove **Details** and maybe move the sentence about the storage systems to the **Overview**.